### PR TITLE
Paginations

### PR DESCRIPTION
**Pull request by hghanbari**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
```
Comment (lencodes) [inline .idea/.gitignore:8]: I don't think this is supposed to be a part of this pull request, what is this file, how and why was it created?

File: src/index.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { DownloadCard, HorizontalImageCard, ImageAndDetailsCard, RichContentCard } from "./components/card";
 import { Container } from "./components/container/Container";
 import { Breadcrumbs } from "./components/denhaag-wrappers/breadcrumbs/Breadcrumbs";
+import {Paginations} from "./components/denhaag-wrappers/paginations/Paginations"
```
Comment (lencodes) [inline src/index.ts:4]: This was not prettified.

File: src/index.ts [hunk 3]
```diff
@@ -40,5 +41,6 @@ export {
   PrivateRoute,
   StatusSteps,
   PrimaryTopNav,
+  Paginations,
```
Comment (lencodes) [inline src/index.ts:44]: It's _Pagination_ and not _PaginationS_.

File: src/components/denhaag-wrappers/paginations/Paginations.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.denhaag-pagination {
+  display: flex;
+  justify-content: center;
+  padding: var(--nlportal-space-inline-md);
+}
+
+.denhaag-pagination__link {
+  padding-right: var(--nlportal-space-inline-md);
+  text-decoration: none;
+}
```
Comment (lencodes) [inline src/components/denhaag-wrappers/paginations/Paginations.css:10]: The styling of this Pagination wrapper is not the same as on the storybook. I think it has to do with this css being (much) too little.

Comment (lencodes) [reply src/components/denhaag-wrappers/paginations/Paginations.css:10]: Tip: the best way to find the right css is by searching within their repository and to use that.

File: src/components/denhaag-wrappers/paginations/Paginations.tsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
+/**
```
Comment (lencodes) [inline src/components/denhaag-wrappers/paginations/Paginations.tsx:1]: The functionality of this Pagination component is much smaller than the one in the Den Haag storybook. Right now, I think we're missing:

- Link highlighting;
- Current page indicator;
- Disabled page indicators;
- Disabled prev / next buttons;
- ... 

For the full functionality you can check the storybook!

**Review by lencodes — CHANGES_REQUESTED** [review]
A good first version @hghanbari. I do have some comments.

There's a `.idea` folder added to this pull request, I'm unsure what the files are and I doubt it should be part of this pull request.

**Comment by hghanbari** [comment]
Dit component werkt nog niet optimaal.
Dit komt doordat de styling van den haag niet werkt.
9:25 AM
@remko48 is in overleg met Den-Haag

**Pull request closed without merging**